### DISCUSSION TITLE
how to create an installer for a simple revit addin via template?

### DISCUSSION THREAD

**Discussion by benkoshy** [Nice3point/RevitTemplates#55] [Q&A]
Hi there
First thank you for your project, for creating it. 
I am trying to build a ‘hello world’ plugin
I want to give this dll to another user so that they can “install” the plugin and have it work on their computer.
I followed the instructions here: https://github.com/Nice3point/RevitTemplates/wiki/Installation#install-nuke-global-tool
And installed nuke. Then I ran:

```sh
nuke :setup
```
Incidentally this is what came up.
 
```sh
NUKE Global Tool 🌐 version 8.0.0 (Windows,.NETCoreApp,Version=v6.0)
Let's setup a new build!
🌳  Root directory:           C:\Users\benko\source\repos\RevitAddInTest
How should the project be named? (_build): Failed to read input in non-interactive mode.
```

I feel I am missing something major? 

Any pointers on how I can build an installer for the revit addin?

**Comment by Nice3point** [comment]
Hi, you need to use the Solution template https://github.com/Nice3point/RevitTemplates/wiki/Templates#revit-addin-solution

Solution template contains a Readme file, with MSI build instructions
https://github.com/Nice3point/RevitTemplates/blob/main/samples/MultiProjectSolution/Readme.md

Please dont run `nuke :setup` command, the Nuke project already exists in the Solution root directory
